h/events.py

```diff
@@ -24,3 +27,6 @@ def __init__(self, request, annotation, annotation_dict):
         self.request = request
         self.annotation = annotation
         self.annotation_dict = annotation_dict
+
+    def __eq__(self, other):
+        return self.__dict__ == other.__dict__
```

Comment:
This method is not being called anywhere at the moment. We should either remove it, or use it in the tests (and also test the equality method itself).

Reply:
I've been looking into this further and it looks like making classes robustly value-like in Python is actually a bit fiddly and there isn't a convenient mixin or decorator in the stdlib that I can just pull in that I can see. 🙁

For the moment I've gone with something like your original suggestion which is just to compare the field values of the actual and expected events directly in the test.